pom.xml

```diff
@@ -134,6 +134,27 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-minikdc</artifactId>
+            <version>${hadoop.version}</version>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <!-- exclude this from hadoop minikdc as the minikdc depends on
+                    the apacheds-jdbm1 bundle, which is not available in maven central-->
+                    <groupId>org.apache.directory.server</groupId>
+                    <artifactId>apacheds-jdbm1</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <!-- add this to satisfy the dependency requirement of jdbm-->
+            <groupId>org.apache.directory.server</groupId>
+            <artifactId>apacheds-jdbm</artifactId>
+            <version>2.0.0-M2</version>
+            <scope>test</scope>
```

Comment:
@Ishiihara Actually, I think this just has some typos. In the exclusion, it should be `org.apache.directory.jdbm`, not `org.apache.directory.server`. In the dependency, you need the same change and the artifact ID is missing a 1 at the end, i.e. it should be `apacheds-jdbm1`.

With those changes, it builds and the tests pass.